### ISSUE TITLE
Bug / Compat : Over-enchanting Adaptive Thermal Barrier

### ISSUE THREAD

**Issue by AzureMonument** [anIceyGirl/SurvivalOverhaul#35]
So, I've got a simple well-made blaze farm in the nether, and a trading hall I've been using to get enchanted books. Now, one of the mods I have in conjunction with this is a mod that allows me to over-enchant books. I've recently gotten four books of Adaptive Thermal Resistance and enchanted each piece of my gear.

![2021-02-24_15 09 43](https://user-images.githubusercontent.com/43318292/109060001-dda89000-76b2-11eb-87ac-0f78d834f724.png)

Now, you may be able to tell that I am in the nether. You may also be able to tell that I am slowly freezing to death (shaking snowflake, though I haven't taken damage yet).

I am roughly 80% certain this is not an intended feature.
(losing fire resistance seems to have made it so I'm only at the snowflake (no shake))

EDIT: I just realized the enchantment is called 'adaptive thermal barrier', not resistance. :P 
EDIT 2: Also, I'm still occasionally getting the shaky snowflake in the nether, even without fire res.

**Comment by anIceyGirl** [comment]
Yeah, it does seem like the enchantment can be a little over-zealous. I'm assuming you're using the latest version of the mod, so I'll try looking into it when I get the chance.

**Comment by Jusey1** [comment]
I think what is happening with the Adaptive enchantment is that it is detecting that you are HOT (due to being in the Nether) and thus all four enchantments of the mod is applying a COLD effect to you. They all work at full power instead of using some of their power until you're neutral.
